### ISSUE TITLE
Doesn't annotate non-fixable issues when using --fix

### ISSUE THREAD

**Issue by keironlowe-edriving** [sibiraj-s/action-eslint#10]
When using `--fix` in `eslint-args` it correctly fixes the issues it can fix, but then it no longer adds an annotation for those that it can't fix. Appears to be an issue when committing the fixes afterwards. I'm using `EndBug/add-and-commit@v9` and when I disable that step out, the issues it can fix aren't annotated (so they are being fixed), but the issue it can't fix is correctly annotated.

The issue is still detected by eslint, just the annotation isn't added.

<img width="1044" alt="image" src="https://user-images.githubusercontent.com/73830244/225363955-1af7ae69-c876-47d6-9150-90d6c2eec6de.png">

```
- name: Run ESLint
  uses: sibiraj-s/action-eslint@v2
  with:
    eslint-args: '--fix'
    extensions: 'js,ts,tsx,html'
    annotations: true

- name: Commit changes
  uses: EndBug/add-and-commit@v9
  with:
    message: 'ESLint Fixes'
```

**Comment by keironlowe-edriving** [comment]
I'm closing this issue. After some more investigation, it appears that the annotations are correctly added from the action, but because we're adding a new commit, the annotations are added to the original commit, so they don't exist on the new one. Hence they don't show on the PR code review screen.

**Comment by keironlowe-edriving** [comment]
For anyone else who runs into the same issue, I worked around this issue by using a custom token/bot account for the `actions/checkout` step. By default, the token that's used doesn't trigger the workflow on any new commits created by the workflow. If you use a custom token then the workflow will be triggered for the new commit, and annotations correctly displayed.

**Comment by sibiraj-s** [comment]
Thanks @keironlowe-edriving . 

Sorry for the delay in response. Glad you have figured out the issue.
